tempto-examples/build.gradle

```diff
@@ -41,6 +41,7 @@ jar {
 
 shadowJar {
     version = ''
+    zip64 = true
```

Comment:
@ZacBlanco on gradle build, I am getting error 

```
Execution failed for task ':tempto-examples:shadowJar'.
> shadow.org.apache.tools.zip.Zip64RequiredException: archive contains more than 65535 entries.
```


<img width="795" alt="image" src="https://github.com/user-attachments/assets/e8b258db-c7b1-4648-a089-18a46abe3566">

Here, ShadowJar plugin is creating a JAR file that exceeds the limit of 65,535 entries. This might due to the ZIP format limitation that ShadowJar uses by default. [Link](https://github.com/GradleUp/shadow/issues/107)

Reply:
Got it, thanks for the explanation. This is fine then I guess. We were probably close to the limit and the newer version pushed us over? It would be good to know the number of entries before/after this change, but I don't think it's worth trying to limit